package.json

```diff
@@ -21,7 +21,8 @@
     "g:s": "ng g service --app=1",
     "package:build": "node scripts/build.js",
     "package:make-local-tgz": "node scripts/make-local-tgz.js",
-    "package:release": "node scripts/release.js"
+    "package:release": "node scripts/release.js",
+    "prepackage:release": "npm run test -- --watch=false --progress=false --browsers=ChromeHeadless"
```

Comment:
By default, adding "pre" to a script name will automatically run that script before. So when you run `npm run package:release` it will first execute `npm run prepackage:release`. Basically ensuring that our tests pass before we release the package

Reply:
NIce one, good to know